Sources/Plasma/PubUtilLib/plGLight/plDirectShadowMaster.cpp

```diff
@@ -66,10 +66,8 @@ plDirectShadowMaster::plDirectShadowMaster()
 
 plDirectShadowMaster::~plDirectShadowMaster()
 {
-    fIsectPool.SetCount(fIsectPool.GetNumAlloc());
-    int i;
-    for( i = 0; i < fIsectPool.GetCount(); i++ )
-        delete fIsectPool[i];
+    for (plBoundsIsect* isect : fIsectPool.pool())
+        delete isect;
```

Comment:
This seems a little ugly to me. Since this is a small PR, maybe it would be better to go ahead and convert these naked pointers to either an `hsRef` or `std::unique_ptr`?